bridge/workflows/send_payouts.py

```diff
@@ -63,19 +65,35 @@ def mark_permanent_failure(request, error_message):
 	sender = TransactionSender(network)
 	await sender.init(vault_connector)
 
+	# in stake mode, requests must be processed in order to ensure stake attribution is always correct
+	can_skip = StrategyMode.STAKE != execution_context.strategy_mode
+
 	for request in requests_to_send:
 		conversion_rate_calculator = conversion_rate_calculator_factory.try_create_calculator(request.transaction_height)
 		if not conversion_rate_calculator:
-			mark_skipped(request, 'missing data')
+			mark_skipped(request, f'missing data at height {request.transaction_height}')
 			continue
 
 		logger.info('processing payout: %s:%s', request.transaction_hash, request.transaction_subindex)
-		conversion_function = conversion_rate_calculator.to_conversion_function(is_unwrap_mode)
+		conversion_function = conversion_rate_calculator.to_conversion_function(execution_context.is_unwrap_mode)
+		send_result = None
 		try:
-			send_result = await _send_payout(sender, network, request, conversion_function, fee_multiplier)
+			prepare_send_result = prepare_send(network, request, conversion_function, fee_multiplier)
+			if prepare_send_result.error_message:
+				send_result = TrySendResult(True, None, None, None, prepare_send_result.error_message)
+			else:
+				(is_exceeded, amount_remaining) = is_daily_limit_exceeded(network, database, prepare_send_result.transfer_amount)
+				if is_exceeded:
+					mark_skipped(request, f'daily limit exceeded ({amount_remaining} < {prepare_send_result.transfer_amount})')
+					if not can_skip:
```

Comment:
For non-STAKE (WRAP and SWAP) modes, payments exceeding the daily limit are sent. I assume we should add an else statement with continue to process to the next request.